go/.htaccess

```diff
@@ -8,7 +8,8 @@ RedirectMatch "/go/(([1-9][0-9])([.]?)([0-9]))/developer-help-(mobile|packages)(
 # Download redirects for keyboard permalinks (TODO: these three rules need refresh)
 
 # download-kmp
-RedirectMatch "/go/keyboard/([^/?]+)/download/kmp$" "/keyboards/download?id=$1&platform=windows&mode=standalone"
+# Modified to match download-package below
+RewriteRule "^keyboard/([^/]+)/download/kmp$" "package/download.php?type=keyboard&id=$1" [END,QSA]
 
 #download-exe
 RedirectMatch "/go/keyboard/([^/?]+)/download/exe$" "/keyboards/download?id=$1&platform=windows&mode=bundle"
```

Comment:
What about this rule and download-js? They also seem to be wrong?

Reply:
For .exe, look at the pattern generated e.g. by https://keyman.com/keyboards/install/khmer_angkor, and use that: https://downloads.keyman.com/windows/stable/17.0.331/keyman-setup.khmer_angkor.exe

Reply:
download-js should go to downloads.keyman.com/keyboards/name/version/name.js, e.g. https://downloads.keyman.com/keyboards/thamizha_bamini/2.2/thamizha_bamini.js

Reply:
These will be split into #495